Assets/Scripts/PlayerInventory.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+
+public class PlayerInventory : MonoBehaviour
+{
+    public LayerMask ignoreLayers = 1 << 4;
+    public float pickUpRange = 2f;
+    public Transform hand;
+    public Vector3 inHandSize = new Vector3(0.2f, 0.2f, 0.2f);
+    public Ammunition ammunition;
+
+    void Update()
+    {
+        if (ammunition)
+        {
+            ammunition.transform.position = hand.position;
+        }
+        else if (Input.GetButtonDown("Fire2")
+                && Physics.Raycast(
+                    Camera.main.ScreenPointToRay(Input.mousePosition),
+                    out var hitInfo,
+                    pickUpRange,
+                    ~ignoreLayers))
```

Comment:
It's better to invert the value of the variable concept instead of inverting it through code.

Suggest renaming this variable to something like `pickUpLayerMask` and removing this bitwise-unary-not operator.

[G29: Avoid Negative Conditionals](https://moderatemisbehaviour.github.io/clean-code-smells-and-heuristics/general/g29-avoid-negative-conditionals.html)